specification/liftrweightsandbiases/Liftr.WeightsAndBiases.Management/tspconfig.yaml

```diff
@@ -32,6 +32,11 @@ options:
     generate-fakes: true
     head-as-boolean: true
     inject-spans: true
+  "@azure-tools/typespec-java":
+    package-dir: "azure-resourcemanager-weightsandbiases"
```

Comment:
Do we want to override `service-dir` as well?

Reply:
thanks. yes we do. updated.